add translation for get-started-calico (#1565)
spidernet-io/spiderpool: docs/usage/get-started-calico.md

# Calico Quick Start

**English** | [**简体中文**](./get-started-calico-zh_CN.md)

> SpiderPool is able to provide static IPs to deployment, StatefulSet, and other types of applications in underlay networks. In this page, we'll introduce how to build a complete Underlay network environment in Calico + BGP mode, and integrate it with SpiderPool to enable fixed IP addresses for applications. This solution meets the following requirements:

Reviewer: ```suggestion
SpiderPool is able to provide static IPs to deployments, StatefulSets, and other applications in underlay networks. In this page, we'll introduce how to build a complete underlay network environment in Calico + BGP mode, and integrate it with SpiderPool to enable fixed IP addresses for applications. This solution meets the following requirements:
```
哪种写法是对的？
- Spiderpool
- spiderpool
- SpiderPool

Author: 已统一：Spiderpool

* Assign static IP addresses to applications

> * IP pools dynamically scale based on replica counts

Reviewer: ```suggestion
* Scale IP pools dynamically based on replica counts
```
这个词也是，哪个是对的？
- IP pool
- IPPool
- IPpool

Author: IP pool

> * External clients outside the cluster can access applications without their IPs

Reviewer: ```suggestion
* Enable external clients outside a cluster to access applications without their IPs
```
3 个句式需要一致：v + n

Author: 已改
